### PR TITLE
Add reading time option

### PR DESCRIPTION
**Pull request by ribice**
Add option to include reading time.

How it looks like:

screenshot at aug 07 12-08-24

The icon does look a bit out of place - borders should be thinner. Perhaps same source as other icons should be used.

### CODE REVIEW

File: layouts/partials/svg/icons.html [hunk 1]
```diff
@@ -34,5 +34,8 @@
...
+  <path d="M15.129 7.25c-0.138-0.239-0.444-0.321-0.683-0.183l-4.92 2.841-3.835-2.685c-0.226-0.158-0.538-0.103-0.696 0.123s-0.103 0.538 0.123 0.696l4.096 2.868c0.001 0.001 0.002 0.001 0.002 0.002 0.009 0.006 0.018 0.012 0.027 0.017 0.002 0.001 0.004 0.003 0.006 0.004 0.009 0.005 0.018 0.010 0.027 0.015 0.002 0.001 0.004 0.002 0.006 0.003 0.010 0.005 0.020 0.009 0.031 0.014 0.006 0.003 0.013 0.005 0.019 0.007 0.004 0.001 0.008 0.003 0.013 0.005 0.007 0.002 0.014 0.004 0.021 0.006 0.004 0.001 0.008 0.002 0.012 0.003 0.007 0.002 0.014 0.003 0.022 0.005 0.004 0.001 0.008 0.002 0.012 0.002 0.007 0.001 0.014 0.002 0.021 0.003 0.005 0.001 0.010 0.001 0.015 0.002 0.006 0.001 0.012 0.001 0.018 0.002 0.009 0.001 0.018 0.001 0.027 0.001 0.002 0 0.004 0 0.006 0 0 0 0-0 0-0s0 0 0.001 0c0.019 0 0.037-0.001 0.056-0.003 0.001-0 0.002-0 0.003-0 0.018-0.002 0.036-0.005 0.054-0.010 0.002-0 0.003-0.001 0.005-0.001 0.017-0.004 0.034-0.009 0.050-0.015 0.003-0.001 0.006-0.002 0.008-0.003 0.016-0.006 0.031-0.012 0.046-0.020 0.004-0.002 0.007-0.004 0.011-0.006 0.005-0.003 0.011-0.005 0.016-0.008l5.196-3c0.239-0.138 0.321-0.444 0.183-0.683z"/>
```
Comment (MunifTanjim) [inline layouts/partials/svg/icons.html]: Use this instead:

```svg
<circle cx="12" cy="12" r="10"/>
<polyline points="12 6 12 12 15 15"/>
```

File: layouts/partials/entry/meta.html [hunk 2]
```diff
@@ -7,6 +7,14 @@
       {{- .Date.Format ( or .Site.Params.dateFormat "2006, Jan 02" ) -}}
     </time>
   </span>
+  <span class='read-time'>
+    {{- if not .Site.Params.hideReadTime -}}
+    {{ partial "svg/icons" "clock" }}
+     <time class='reading-time'>
```
Comment (MunifTanjim) [inline layouts/partials/entry/meta.html]: We don't need `time` tag here. A simple `span` would do. :smiley:

File: exampleSite/config.toml [hunk 3]
```diff
@@ -20,6 +20,7 @@ preserveTaxonomyNames = true
   customCSS = ["css/custom.css"]
   customJS = ["js/custom.js"]
   hideSocialMenu = false
+  hideReadTime = false
```
Comment (MunifTanjim) [inline exampleSite/config.toml]: Could you please change this option to `showReadingTime` with default value to `false`?
The current default behavior will be preserved that way.

**Comment by ribice** [comment]
Applied changes and rebased. You should provide translation for Bengali.

Just curious, where do you get icons from? This one looks good.

**Comment by MunifTanjim** [comment]
@ribice It's [feather](https://github.com/colebemis/feather) icons...

**Pull request merged**